daras_ai_v2/base.py

```diff
@@ -410,7 +414,7 @@ def _render_author_as_breadcrumb(self, *, is_example: bool, is_root_example: boo
         if not workspace:
             return
 
-        with gui.breadcrumbs(divider="▶"):
+        with gui.breadcrumbs(divider=""):
```

Comment:
i tried adding the SVG here as the divider - https://getbootstrap.com/docs/5.2/components/breadcrumb/

but it is tricky to get right - SVG has to be encoded correctly and we would need to change the code in `gooey-gui` too to make that work - to remove single quotes we put before adding it to `style=...`. this is a simple workaround for a rare case. we keep icons simple & consistent in html, and avoid complexity of conforming to the data-URI and encoding that is needed.